scripts/meetingbaas.py

```
@@ -213,16 +207,15 @@ async def on_connection_error(transport, error):
-        speed="normal",
     )
     log_and_flush(logging.INFO, f"[TTS] Cartesia TTS initialized with sample_rate={output_sample_rate}, voice_id={voice_id}")
 
     llm = OpenAILLMService(
         api_key=os.getenv("OPENAI_API_KEY"),
-        model="gpt-4-turbo-preview",
+        model="gpt-4.1",
         run_in_parallel=False,
     )
-    log_and_flush(logging.INFO, f"[LLM] OpenAI LLM initialized with model=gpt-4-turbo-preview")
+    log_and_flush(logging.INFO, f"[LLM] OpenAI LLM initialized with model=gpt-4.1")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)